src/js/components/modals/InstallPackageModal.js

```diff
@@ -147,6 +162,25 @@ class InstallPackageModal
     this.setState({ currentTab: "packageInstalled" });
   }
 
+  handleAcceptBetaTerms() {
+    // Set accept terms on config model
+
+    if (this.props.advancedConfig) {
```

Comment:
This method is unnecessary complex. Please adjust the conditions and setState calls to reduce the complexity.